docker/Dockerfile

```diff
@@ -42,17 +45,18 @@ RUN npm ci
 
 # Base arguments: build-time
 ARG NPM_CONFIG_LOGLEVEL=warn
-ARG PORT=3000
+ARG PORT=8787
 
 # Network API endpoints: build-time
 ARG MAINNET_RPC_URL=https://rpc.cheqd.net:443
 ARG TESTNET_RPC_URL=https://rpc.cheqd.network:443
 ARG RESOLVER_URL=https://resolver.cheqd.net/1.0/identifiers/
 
 # Veramo Database configuration: build-time
-ARG DB_CONNECTION_URL
-ARG DB_ENCRYPTION_KEY
-ARG DB_CERTIFICATE
+ARG ENABLE_EXTERNAL_DB
```

Comment:
could we also specify a default value for this variable?

Reply:
Done! ✅